bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,161 @@
+  echo "Error: Output directory '${output_dirpath}' exists, but is not empty"
+  show_help_and_exit
+fi
+if [ -z "${lambda_image}" ]; then
+  echo "Error: Kurtosis Lambda image cannot be empty" >&2
+  show_help_and_exit
+fi
+sanitized_image="$(echo "${lambda_image}" | sed "s|[^${ALLOWED_IMAGE_NAME_CHARS}]||g")"
+if [ "${sanitized_image}" != "${lambda_image}" ]; then
+  echo "Error: Kurtosis Lambda image name '${lambda_image}' doesn't match regex [${ALLOWED_IMAGE_NAME_CHARS}]+" >&2
+  show_help_and_exit
+fi
+
+# =============================================================================
+#                                 Main Code
+# =============================================================================
+# Use language-specific prep script to populate contents of output directory
+if ! mkdir -p "${output_dirpath}"; then
+  echo "Error: Could not create output directory '${output_dirpath}'" >&2
+  exit 1
+fi
+lang_dirpath="${repo_root_dirpath}/${lang}"
+lang_bootstrap_dirpath="${script_dirpath}/${lang}"
+prep_new_repo_script_filepath="${lang_bootstrap_dirpath}/${PREP_NEW_REPO_FILENAME}"
+if ! bash "${prep_new_repo_script_filepath}" "${lang_dirpath}" "${output_dirpath}" "${lambda_image}"; then
+  echo "Error: Failed to prep new repo using script '${prep_new_repo_script_filepath}'" >&2
+  exit 1
+fi
+
+# README file
+output_readme_filepath="${output_dirpath}/${OUTPUT_README_FILENAME}"
+cat <<EOF >"${output_readme_filepath}"
+My Kurtosis Lambda
+=====================
+Welcome to your new Kurtosis Lambda! You can use Example Kurtosis Lambda implementation as a pattern to create your own Kurtosis Lambda.
+
+1. Customize your own Kurtosis Lambda by editing the generated files inside the `/path/to/your/code/repos/kurtosis-lambda/impl` folder
+  1. Follow the bootstrap instructions
+
+EOF
+if [ "${?}" -ne 0 ]; then
+  echo "Error: Could not write README file to '${output_readme_filepath}'" >&2
+  exit 1
+fi
+
+#Initialize the new repo as a Git directory, because running the Kurtosis Lambda depends on it
+if ! command -v git &> /dev/null; then
+    echo "Error: Git is required to create a new Kurtosis Lambda repo, but it is not installed" >&2
+    exit 1
+fi
+if ! cd "${output_dirpath}"; then
+    echo "Error: Could not cd to new Kurtosis Lambda repo '${output_dirpath}', which is necessary for initializing it as a Git repo" >&2
+    exit 1
+fi
+if ! git init; then
+    echo "Error: Could not initialize the new repo as a Git repository" >&2
+    exit 1
+fi
+if ! git add .; then
+    echo "Error: Could not stage files in new repo for committing" >&2
+    exit 1
+fi
+if ! git commit -m "Initial commit" > /dev/null; then
+    echo "Error: Could not create initial commit in new repo" >&2
+    exit 1
+fi
+
+#Runs build script
+scripts_dirpath="${output_dirpath}/${SCRIPTS_DIRNAME}"
+bash "${scripts_dirpath}/${BUILD_FILENAME}"
+
+echo "Bootstrap successful!"
+echo "For next steps, follow the instructions in the README at /path/to/new/README"
```

Comment:
We know the exact path this README is outputted at - it's `output_readme_filepath` - so we should use that instead